### PR TITLE
Page should have a transparent background when `Minimized` is enabled

### PR DESCRIPTION
**Pull request by hqhhuang**
Bug/issue #, if applicable: 103150955

## Summary
Documentation pages should have a transparent background when the prop `enableMinimized` is set to true

## Testing
Steps:
1. Manually verify that the background is set to transparent in both light and dark modes.
2. Verify that no regression is introduced when Minimized mode is not enabled. Make sure to scroll further up when the page is already at the top, and scroll further down when the page is already at the bottom.

### CODE REVIEW

**Comment by hqhhuang** [comment]
@swift-ci test

File: src/styles/base/_typography.scss [hunk 1]
```diff
@@ -21,7 +21,9 @@ html {
 
 body {
   @include font-styles(body);
-  background-color: var(--color-text-background);
+  @include inTargetWeb {
```
Comment (mportiz08) [inline src/styles/base/_typography.scss:24]: You might consider removing this change and simply setting the `--color-text-background` to `transparent` in the place where that variable is being overwritten for IDE targets.

File: src/views/DocumentationTopic.vue [hunk 2]
```diff
@@ -477,6 +481,10 @@ export default {
   }
 }
 
+.background-disabled {
```
Comment (mportiz08) [inline src/views/DocumentationTopic.vue:484]: This change can likely be avoided altogether if we instead simply override the `--color(s)-text-background` variable to be transparent for IDE targets.

**Comment by mportiz08** [comment]
@hqhhuang can we close this now or was there a specific reason for keeping this up as a draft PR?

**Pull request closed without merging**